flink-runtime/src/test/java/org/apache/flink/runtime/rpc/RpcEndpointTest.java

```diff
@@ -51,7 +50,7 @@
 @ExtendWith(TestLoggerExtension.class)
 public class RpcEndpointTest {
 
-    private static final Time TIMEOUT = Time.seconds(10L);
+    private static final Duration TIMEOUT = Duration.ofSeconds(10L);
```

Comment:
This one is only used in a single test. I think, we could move the timeout there...